### PR TITLE
Added package Installation procedure

### PR DESCRIPTION
**Pull request by nitesh585**
please review it :)

### CODE REVIEW

**Review by nitesh585 — COMMENTED** [review]
Please review it :)

File: README.md [hunk 1]
```diff
@@ -3,11 +3,18 @@ A platform for internal tools at Coding Blocks, hosted at https://tools.codingbl
...
+npm is installed with Node.js<br>
+This means that you have to install Node.js to get npm installed on your computer.<br>
+Download Node.js from the official Node.js web site: [https://nodejs.org](https://nodejs.org)
+### Installing yarn
+`npm install -g yarn`
 
-```bash
-yarn dev
-```
+change the directory to `/gondor` project in terminal.
```
Comment (0xnmn) [inline README.md]: remove this

File: README.md [hunk 2]
```diff
@@ -3,11 +3,18 @@ A platform for internal tools at Coding Blocks, hosted at https://tools.codingbl
 
 ## Getting Started
 
-First, run the development server:
+### Installing npm
```
Comment (0xnmn) [inline README.md]: Do not need this.

**Review by 0xnmn — CHANGES_REQUESTED** [review]

**Comment by nitesh585** [comment]
changes have done...please review

**Pull request closed without merging**